lib/cli/src/frameworks/vue3/Button.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <button type="button" :class="classes" @click="onClick" :style="style">{{ label }}</button>
+</template>
+
+<script>
+import './button.css';
+import { reactive, computed } from 'vue';
+
+export default {
+  name: 'my-button',
+
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    primary: {
+      type: Boolean,
+      default: false,
+    },
+    size: {
+      type: String,
+      default: 'medium',
+      validator: function (value) {
+        return ['small', 'medium', 'large'].indexOf(value) !== -1;
+      },
+    },
+    backgroundColor: {
+      type: String,
+    },
+  },
+
+  emits: ['onClick'],
```

Comment:
This may be nit-picking, but should we drop the `on` here? In vue2 the `on` keyword actually had semantic value in certain contexts (jsx, the vnode data interface, etc), and it could possibly throw people off here. Maybe changing it just to `click` (and the template in `Button.stories` to match, of course) would be less potentially misleading 🤷‍♂️ 

But again, I know that's nit-picky

Reply:
I agree with you, i just did transform the vue2 components to vue3 
:+1: